### ISSUE TITLE
Nd4j TF Import issues with Keras BatchNorm

### ISSUE THREAD

**Issue by rnett** [eclipse/deeplearning4j#7964]
Keras's `BatchNorm` has a few issues with tensorflow import (and tensorflow in general).

 * Exported tf nodes all have `is_training` true and don't use `moving_mean` or variance as input.
 * The `moving_mean` etc need to be updatable during training.
   * This is done seperatly by the Keras `fit` method and not part of the graph, see tensorflow/tensorflow#23873, keras-team/keras#6752, and tensorflow/tensorflow#19643
